app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -209,3 +222,14 @@ function PoolPairAPR (props: { symbol: string, apr: number, row: string }): JSX.
     </View>
   )
 }
+
+function DefaultSkeletonLoader (): JSX.Element {
+  const skeletonRow = 3
+  return (
+    <>
+      {Array.from(Array(skeletonRow), (_v, i) => i + 1).map((_x, i) => (
```

Comment:
this is too complex, haha just make it 

const skeletonRow = [1,2,3]

skeletonRow.map((x) => <skeletonrow key={x} />)

Reply:
😅 updated